src/comp/comp.py

```diff
@@ -25,47 +30,67 @@ def __repr__(self):
 # whose name starts with 'D':
 print("Starts with D:")
 a = []
+for i in humans:
+    if i.name.startswith('D'):
+        a.append(i.name)
 print(a)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name ends in "e".
 print("Ends with e:")
 b = []
+for i in humans:
+    if i.name.endswith('e'):
+        b.append(i.name)
 print(b)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with any letter between 'C' and 'G' inclusive.
 print("Starts between C and G, inclusive:")
+# c = [i for i in humans if re.match(r'[C-G, c-g]', str(i))]
 c = []
+for i in humans:
+    if re.match(r'[C-G, c-g]', i.name):
+        c.append(i.name)
 print(c)
 
 # Write a list comprehension that creates a list of all the ages plus 10.
 print("Ages plus 10:")
 d = []
+for i in humans:
+    d.append(i.age + 10)
 print(d)
 
 # Write a list comprehension that creates a list of strings which are the name
 # joined to the age with a hyphen, for example "David-31", for all humans.
 print("Name hyphen age:")
 e = []
+for i in humans:
+    e.append(f'{i.name}-{i.age}')
 print(e)
 
 # Write a list comprehension that creates a list of tuples containing name and
 # age, for example ("David", 31), for everyone between the ages of 27 and 32,
 # inclusive.
 print("Names and ages between 27 and 32:")
 f = []
+for i in humans:
+    if i.age < 33 and i.age > 26:
+        f.append((i.name, i.age))
 print(f)
 
 # Write a list comprehension that creates a list of new Humans like the old
 # list, except with all the names uppercase and the ages with 5 added to them.
 # The "humans" list should be unmodified.
 print("All names uppercase:")
 g = []
+for i in humans:
+    g.append(Human(i.name.upper(), i.age + 5))
 print(g)
 
 # Write a list comprehension that contains the square root of all the ages.
 print("Square root of ages:")
-import math
 h = []
+for i in humans:
+    h.append(i.age ** .5)
```

Comment:
Whoa we got a math wizard up in here.
There is a sqrt method from the math library:
`h = [math.sqrt(human.age) for human in humans]`